pkg/updater/defaults/catalog.json

```diff
@@ -1,44 +1,24 @@
 {
-    "packages": [
-        {
-            "name": "datadog-agent",
-            "package": "datadog-agent",
-            "version": "7.52.0-rc.1.git.15.6c19b17.pipeline.28703603-1",
-            "platform": "linux",
-            "arch": "amd64",
-            "url": "https://d34l0745r4hrvf.cloudfront.net/oci/testing/datadog-agent_7.52.0-rc.1.git.15.6c19b17.pipeline.28703603-1_oci_amd64.tar",
-            "sha256": "35f8be32eb2958408342f13e85878bcd5e591f9ad581c2fee3b77af7efee45c7",
-            "size": 373616640
-        },
-        {
-            "name": "datadog-agent",
-            "package": "datadog-agent",
-            "version": "7.52.0-rc.1.git.15.6c19b17.pipeline.28703603-1",
-            "platform": "linux",
-            "arch": "arm64",
-            "url": "https://d34l0745r4hrvf.cloudfront.net/oci/testing/datadog-agent_7.52.0-rc.1.git.15.6c19b17.pipeline.28703603-1_oci_arm64.tar",
-            "sha256": "390aa03cf93a84e340f4baeea01edc0c5f306350a6794a895917bf28f1e306eb",
-            "size": 321075200
-        },
-        {
-            "name": "datadog-agent",
-            "package": "datadog-agent",
-            "version": "7.52.0-rc.1.git.15.6c19b17.pipeline.28703556-1",
-            "platform": "linux",
-            "arch": "amd64",
-            "url": "https://d34l0745r4hrvf.cloudfront.net/oci/testing/datadog-agent_7.52.0-rc.1.git.15.6c19b17.pipeline.28703556-1_oci_amd64.tar",
-            "sha256": "6b6c7abafbe70fe04edbe2a672dcd97e9b87943513a28ac5ecf09a99aed1c40e",
-            "size": 373657600
-        },
-        {
-            "name": "datadog-agent",
-            "package": "datadog-agent",
-            "version": "7.52.0-rc.1.git.15.6c19b17.pipeline.28703556-1",
-            "platform": "linux",
-            "arch": "arm64",
-            "url": "https://d34l0745r4hrvf.cloudfront.net/oci/testing/datadog-agent_7.52.0-rc.1.git.15.6c19b17.pipeline.28703556-1_oci_arm64.tar",
-            "sha256": "0592615015f877abc568fbf71e128ec20d34ff8f9ca70db8847ccee69bc83152",
-            "size": 321044480
-        }
-    ]
+  "packages": [
+    {
+      "name": "datadog-agent",
+      "package": "datadog-agent",
+      "version": "7.52.0-rc.1.git.15.6c19b17.pipeline.28815219-1",
+      "platform": "linux",
+      "arch": "amd64",
+      "url": "https://dd-agent-macostesting.s3.amazonaws.com/oci/testing/datadog-agent_7.52.0-rc.1.git.15.6c19b17.pipeline.28815219-1_oci_amd64.tar",
```

Comment:
Is this the expected version you want to ship ? That bucket looks weird but maybe it was just an easy place to store temp artifacts at ?

Reply:
We're currently sharing the same bucket for OCI and macOS among other things, so it's scary but expected.

Ultimately this is likely to be revamped as `install.datadog.com` or similar